src/Redux/configuration.js

```diff
@@ -0,0 +1,7 @@
+import { combineReducers } from 'redux';
```

Comment:
Please import `createStore` from `redux` and use it to create a store through the reducer you created in line 5 📝

You could try something like:

    const store = createStore(rootReducer);

Don't forget to export your store as well 🤓

    export default store;